src/components/PageBox.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash/fp'
+import { div } from 'react-hyperscript-helpers'
+import { isRadX } from 'src/libs/brand-utils'
+import colors from 'src/libs/colors'
+import * as Utils from 'src/libs/utils'
+
+
+export const PageBoxVariants = {
+  LIGHT: 'light'
```

Comment:
Nit: I don't think we typically uppercase object keys. Should we take the opportunity to lowercase it here?

```suggestion
  light: 'light'
```

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/3594/commits/f1a1d2a6b20c2846aa231c80ec6f60d39383078a.